ksqldb-engine/src/test/java/io/confluent/ksql/execution/pull/HARoutingTest.java

```diff
@@ -574,11 +435,6 @@ public void forwardingError_errorRow() {
     // Then:
     assertThat(pullQueryQueue.size(), is(0));
     assertThat(Throwables.getRootCause(e).getSuppressed()[0].getMessage(), containsString("Row Error!"));
```

Comment:
This test actually checks that the final exception includes also the exception thrown from the forwarded to node